poetry.lock

```diff
@@ -4051,10 +4051,54 @@ description = "Database Abstraction Library"
 optional = true
 python-versions = ">=3.7"
 files = [
+    {file = "SQLAlchemy-2.0.34-cp310-cp310-macosx_10_9_x86_64.whl", hash = "sha256:95d0b2cf8791ab5fb9e3aa3d9a79a0d5d51f55b6357eecf532a120ba3b5524db"},
```

Comment:
Did you intend to include these wheels here?

Reply:
This must have been added when I updated openai. I don't know why it was ever a dependency but clearly it was already there before I ran the update

Reply:
It's actually a dependency of langchain